simulation/simulation.py

```diff
@@ -99,12 +109,28 @@ def _update_simulation_objects(self):
             # TODO: update threshold position
```

Comment:
Czy te todo to są zaznaczenia do przyszłego łączenia z innymi częściami kodu czy to kod do dopisania w ramach tej części?